metrics/release.go

```diff
@@ -218,8 +262,20 @@ func RegisterNewReleaseDeployment() {
 	ReleaseConcurrentDeploymentsTotal.WithLabelValues().Inc()
 }
 
-// RegisterNewReleaseProcessing register a new Release processing, increasing the number of concurrent processings.
-func RegisterNewReleaseProcessing() {
+// RegisterNewReleaseProcessing registers a new Release processing, adding a new observation for the
+// Release start processing duration and increasing the number of concurrent processings. If either the
+// processingStartTime or the releaseStartTime are nil, no action will be taken.
+func RegisterNewReleaseProcessing(startTime, processingStartTime *metav1.Time, reason, target string) {
+	if startTime == nil || processingStartTime == nil {
+		return
+	}
```

Comment:
Could you add a line break like in the other cases?

Reply:
done